src/commands/createStaticWebApp/DeploymentMethodStep.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../utils/localize";
+import { openUrl } from "../../utils/openUrl";
+import { RepoCreateStep } from "../createRepo/RepoCreateStep";
+import { WorkspaceListStep } from "../createRepo/WorkspaceListStep";
+import { GitHubBranchListStep } from "./GitHubBranchListStep";
+import { GitHubOrgListStep } from "./GitHubOrgListStep";
+import { GitHubRepoListStep } from "./GitHubRepoListStep";
+import { IStaticWebAppWizardContext } from "./IStaticWebAppWizardContext";
+
+export type DeploymentMethods = 'local' | 'existingRepo' | 'learnMore';
+
+export class DeploymentMethodStep extends AzureWizardPromptStep<IStaticWebAppWizardContext> {
+    public async prompt(wizardContext: IStaticWebAppWizardContext): Promise<void> {
+        const localCode: IAzureQuickPickItem<DeploymentMethods> = { label: localize('local', '$(cloud-upload) Publish local code to a new GitHub repository'), data: 'local' };
+        const existingRepo: IAzureQuickPickItem<DeploymentMethods> = { label: localize('existingRepo', '$(github) Use existing GitHub repository'), data: 'existingRepo' };
+
+        const placeHolder: string = localize('createMethod', "How do you want to create a static web app?");
+        const learnMore: IAzureQuickPickItem<DeploymentMethods> = { label: localize('learnMore', '$(book) Learn more...'), data: 'learnMore' };
+        let pick: DeploymentMethods;
+
+        do {
+            pick = (await ext.ui.showQuickPick([localCode, existingRepo, learnMore], { placeHolder, suppressPersistence: true })).data;
+            if (pick === learnMore.data) {
+                await openUrl('https://aka.ms/AAaee7c');
```

Comment:
Not sure the link you picked is super helpful. I think I'd prefer you write up some of your own documentation and stick it in our wiki because I'm not sure you're going to find an existing doc specific to our scenario.

Reply:
Will start working on this component now (I didn't forget 😁)

Reply:
Yeah I mean the advantage of an aka.ms link is that you can change it after the PR is merged (but that doesn't mean put it off forever)

Reply:
https://github.com/microsoft/vscode-azurestaticwebapps/wiki/Creating-a-Static-Web-App